### PR TITLE
lesson_10_mihaela_vaida

### PR DESCRIPTION
**Pull request by MihaelaGit**
:)

### CODE REVIEW

File: lesson_10/lesson_10_mihaela_vaida/obj_properties_mihaela.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+class Student(object):
+
+    def __init__(self, first_name, last_name, age):
+        self.first_name = first_name
+        self.last_name = last_name
+        if type(age) is not int:
```
Comment (mariuspodean) [inline lesson_10/lesson_10_mihaela_vaida/obj_properties_mihaela.py]: Let's go one step further and try not having duplicated code

Comment (MihaelaGit) [reply lesson_10/lesson_10_mihaela_vaida/obj_properties_mihaela.py]: I think  I made it 
Property age of Student @age.setter def age(self, age: {setter, __lt__}) -> Any

**Review by zoldirazvan — COMMENTED** [review]
Hi,
I have exactly the same solution, but I still don't get it: why self._age and self.age give us such different results? Somehow Python thinks that is a private variable and should not be changed? Or is related to internal usage only of this kind of variables? Please give me a hint< i'm going crazy here.

**Comment by MihaelaGit** [comment]
Yes, it’ s the privacy of any _variable :) 
This link helped me : https://hackernoon.com/understanding-the-underscore-of-python-309d1a029edc

**Review by mariuspodean — APPROVED** [review]

**Pull request merged**